addons/a11y/src/a11yRunner.ts

```diff
@@ -68,15 +70,20 @@ const run = async (storyId: string) => {
 
 /** Returns story parameters or default ones. */
 const getParams = (storyId: string): A11yParameters => {
-  const { parameters } = globalWindow.__STORYBOOK_STORY_STORE__.fromId(storyId) || {};
-  return (
-    parameters.a11y || {
-      config: {},
-      options: {
-        restoreScroll: true,
-      },
+  const __STORYBOOK_STORY_STORE__ = root && root.__STORYBOOK_STORY_STORE__;
```

Comment:
when is `root` falsey?

Reply:
it should not be - will look at and fix (think it told me it might be, but it should not no)

Reply:
fixed

Reply:
Won't work then

This is actually a good example of why I wanted a package named "global" out

Here jest only will allow us to extend the mock with the inferred "global"

Reply:
Or rather - it won't allow the other way around